service-worker.js

```diff
@@ -0,0 +1,21 @@
+var cacheName = '230-bus-page';
+var filesToCache = ['/', '/index.html'];
+self.addEventListener('install', function(e) {
+  console.log('[ServiceWorker] Install');
+  e.waitUntil(
+    caches.open(cacheName).then(function(cache) {
+      console.log('[ServiceWorker] Caching app shell');
+      return cache.addAll(filesToCache);
+    })
+  );
+});
+self.addEventListener('activate', event => {
+  event.waitUntil(self.clients.claim());
+});
+self.addEventListener('fetch', event => {
```

Comment:
@jakub-g I think this answers your question. 

I went with the most basic approach, meaning that cached version will be served if found. 

For 2 basic reasons: 
- Its blazing fast and simple.
- I assume the content will rarely if ever change. 

There are more "complex" strategies such as networkFirst where we can check if a new version is on the server, but this can be slow if internet speed is bad. 

This is a good read: https://developers.google.com/web/ilt/pwa/introduction-to-progressive-web-app-architectures